documentation/src/main/asciidoc/topics/code_examples/ConfigureTransportDefault.java

```diff
@@ -1,7 +1,6 @@
 GlobalConfiguration globalConfig = new GlobalConfigurationBuilder().transport()
         .defaultTransport()
         .clusterName("qa-cluster")
-         // Use default JGroups stacks with the addProperty() method.
-        .addProperty("configurationFile", "default-jgroups-tcp.xml")
+        .addProperty("configurationFile", "default-jgroups-udp.xml") <1>
         .machineId("qa-machine").rackId("qa-rack")
```

Comment:
maybe remove `machineId` and `rackId` since they aren't required for the example. WDYT?